api/KHR_parallel_shader_compile.json

```diff
@@ -0,0 +1,52 @@
+{
+  "api": {
+    "KHR_parallel_shader_compile": {
```

Comment:
```suggestion
    "KHRParallelShaderCompile": {
```

Reply:
https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/renderer/modules/webgl/khr_parallel_shader_compile.idl

Reply:
Web devs use this as "KHR_parallel_shader_compile", though, so that's what I've been going with for WebGL extensions.

Reply:
See e.g. https://wiki.developer.mozilla.org/en-US/docs/Web/API/ANGLE_instanced_arrays

Reply:
@jpmedley I think this stands correct as is. See also the answers Geoff and Ken via email. Ken wrote:

> Please follow the IDL in the extension's spec when writing the MDN documentation

I did that here.

Reply:
It sounds like this was resolved and there are no outstanding issues here. @jpmedley, please comment if you disagree, otherwise I'll merge this in the next couple of days.

Reply:
I'm still trying to get clarity and I think you can ignore the quote in Florian's comment. I keep asking about an interface and they keep answering as though I'm asking about a string.

Reply:
OK. Thank you for continuing to investigate, @jpmedley. I've labeled this as not ready and I'll await an update.

Reply:
Just keep in mind that MDN so far documents **all** WebGL extensions according to their spec names (and that is with underscores): https://wiki.developer.mozilla.org/en-US/docs/Web/API/WebGL_API/Using_Extensions#Extension_list

Reply:
@jpmedley note the `NoInterfaceObject` in https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/renderer/modules/webgl/khr_parallel_shader_compile.idl. The interface objects for these extensions aren't exposed, making it almost inconsequential what the names of the interfaces are. See https://github.com/mdn/browser-compat-data/issues/6540#issuecomment-677427482 for some ways to observe the name.

I say almost, because it's not unheard of to use `something.toString()` and parsing the "[object InterfaceName]" string as a form of type testing when `something instanceof InterfaceName` cannot work. That doesn't seem super likely in this case, though, since you'd have to have a collection of extensions and not be sure what their types are for this to make sense.